docs/execplans/1-2-4-model-handshake-readiness.md

```
@@ -349,44 +353,45 @@ ______________________________________________________________________
 1. **TLC passes:** Run `make tlc-handshake` and confirm output contains
    "Model checking completed. No error has been found."
 
-2. **Invariants documented:** Verify `MxdHandshake.tla` contains comments
+1. **Invariants documented:** Verify `MxdHandshake.tla` contains comments
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)